### PR TITLE
[System] rework metric dashboards

### PR DESCRIPTION
**Pull request by drewdaemon**
## What does this PR do?

Reworks the metric system overview and host overview dashboards to use Lens. 

Also leverages Lens features to represent current system state more reliably and clearly (especially, uses Lens Last value function instead of TSVB "Last value" mode to resolve https://github.com/elastic/integrations/issues/1437).

## Checklist

- [x] I have reviewed [tips for building integrations](https://github.com/elastic/integrations/blob/main/docs/tips_for_building_integrations.md) and this pull request is aligned with them.
- [x] I have verified that all data streams collect metrics or logs.
- [x] I have added an entry to my package's `changelog.yml` file.
- [x] I have verified that Kibana version constraints are current according to [guidelines](https://github.com/elastic/elastic-package/blob/master/docs/howto/stack_version_support.md#when-to-update-the-condition).

## Related issues

- Closes https://github.com/elastic/integrations/issues/1437
- Relates https://github.com/elastic/integrations/issues/4868

#### Before
overview_before

#### After

system-overview-light

#### Before
host_overview_before

#### After
host-overview-light

### CODE REVIEW

**Comment by drewdaemon** [comment]
Let's keep this open!

**Comment by drewdaemon** [comment]
Keep this open

File: packages/system/kibana/dashboard/system-79ffd6e0-faa0-11e6-947f-177f697178b8.json [hunk 1]
```diff
@@ -1,20 +1,20 @@
...
         "kibanaSavedObjectMeta": {
             "searchSourceJSON": {
                 "filter": [],
-                "highlightAll": true,
                 "query": {
                     "language": "kuery",
-                    "query": ""
-                },
-                "version": true
+                    "query": "host.name:\"docker-fleet-agent\""
```
Comment (dej611) [inline packages/system/kibana/dashboard/system-79ffd6e0-faa0-11e6-947f-177f697178b8.json:9]: Why this filter set here? Is it meant to have a single host on the dashboard?

I see there's a table with this host in the `System overview` as a table, so I assume other hosts should be available as well?

File: packages/system/kibana/dashboard/system-Metrics-system-overview.json
Comment (dej611) [inline packages/system/kibana/dashboard/system-Metrics-system-overview.json:1]: Is the file name meant to have a capital `M`? 

`packages/system/kibana/dashboard/system-Metrics-system-overview.json`

File: packages/system/changelog.yml [hunk 2]
```diff
@@ -1,4 +1,9 @@
 # newer versions go on top
+- version: "1.41.0"
```
Comment (tommyers-elastic) [inline packages/system/changelog.yml:2]: wonder if this warrants a bump to 2.0.0?

**Review by tommyers-elastic — APPROVED** [review]
this looks great, thanks! without reviewing the reams of JSON and just looking at the screenshots, this gets a +1 from me.

File: packages/system/kibana/dashboard/system-79ffd6e0-faa0-11e6-947f-177f697178b8.json (hunk 1 above)
Comment (drewdaemon) [reply packages/system/kibana/dashboard/system-79ffd6e0-faa0-11e6-947f-177f697178b8.json:9]: Good catch. I'll look into this.

**Review by fearful-symmetry — APPROVED** [review]
Looks good, but the one thing that worries me is the `Load` metric. This is only available on non-Windows OSes, so Windows users just get a blank UI item. However, I'm not sure we have the capability to just say "remove the visualization if there's no data"

File: packages/system/kibana/dashboard/system-Metrics-system-overview.json
Comment (drewdaemon) [reply packages/system/kibana/dashboard/system-Metrics-system-overview.json:1]: I agree that it's a little weird, but the filename is automatically generated based on the SO ID. I think that changing these IDs can cause problems with package upgrades, so I don't want to touch it.

**Comment by drewdaemon** [comment]
> I'm not sure we have the capability to just say "remove the visualization if there's no data," hence why I'm approving this. If we can do that and no one told me, we definitely should.

We don't have that capability. It's an interesting feature request. I think our answer today is: create two dashboards. But, I'm going to raise this with the team to see if we are tracking an issue.

File: packages/system/kibana/dashboard/system-79ffd6e0-faa0-11e6-947f-177f697178b8.json [hunk 3]
```diff
@@ -55,1153 +54,1337 @@
...
+                                    },
+                                    "type": "palette"
+                                },
+                                "showBar": true
+                            }
                         },
-                        "title": "",
-                        "type": "metrics",
-                        "uiState": {}
+                        "title": "TSVB visualization (converted)",
```
Comment (efd6) [inline packages/system/kibana/dashboard/system-79ffd6e0-faa0-11e6-947f-177f697178b8.json:369]: Is it worth cleaning these up?

**Review by efd6 — APPROVED** [review]

File: packages/system/kibana/dashboard/system-79ffd6e0-faa0-11e6-947f-177f697178b8.json [hunk 4]
```diff
@@ -55,1153 +54,1337 @@
...
+                                            "columns": {
+                                                "70be4197-83b0-44ae-9f56-11bea599cda6": {
+                                                    "customLabel": true,
+                                                    "dataType": "number",
+                                                    "filter": {
+                                                        "language": "kuery",
+                                                        "query": "system.memory.actual.used.bytes: *"
+                                                    },
+                                                    "isBucketed": false,
+                                                    "label": "Memory usage",
```
Comment (muthu-mps) [inline packages/system/kibana/dashboard/system-79ffd6e0-faa0-11e6-947f-177f697178b8.json:873]: nit: The labels of `memory usage` and `memory usage in percentage` has a different case. Can we update the labels with similar case in all the places?

File: packages/system/kibana/dashboard/system-79ffd6e0-faa0-11e6-947f-177f697178b8.json (hunk 3 above)
Comment (drewdaemon) [reply packages/system/kibana/dashboard/system-79ffd6e0-faa0-11e6-947f-177f697178b8.json:370]: Done in https://github.com/elastic/integrations/pull/6743/commits/281b3e966d1867f51c6fdd02c08bcb17b1455413

Comment (efd6) [reply packages/system/kibana/dashboard/system-79ffd6e0-faa0-11e6-947f-177f697178b8.json:370]: I still see four instances of "TSVB visualization (converted)".

**Review by muthu-mps — APPROVED** [review]
LGTM!

**Comment by drewdaemon** [comment]
The screenshots I added here are quite large. Do we have any guidelines on screenshot size? 

cc @efd6 @muthu-mps

**Review by dej611 — APPROVED** [review]

**Pull request merged**